src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx

```diff
@@ -135,10 +80,25 @@ const DocumentUploader = () => {
         request. Please send us all the documents listed so we can make a
         decision about your request.
       </p>
-      <FileList files={files} onClick={onDeleteClick} />
+      {state.submissionPending ? (
+        <va-loading-indicator label="Loading" message="Sending your files..." />
+      ) : (
+        <FileList files={state.files} onClick={onDeleteClick} />
+      )}
+      {state.successMessage ? (
+        <va-alert
+          background-only
+          close-btn-aria-label="Close notification"
+          show-icon
+          status="success"
+          visible
+        >
+          <div>Your files have been uploaded</div>
```

Comment:
Shouldn't the text be `Your documents were successfully uploaded.`? That's what I'm seeing in the prototype

Reply:
@jerekshoe Good catch, updated this